nautobot/extras/forms/mixins.py

```diff
@@ -370,6 +370,12 @@ def _append_relationships(self):
                 field_name = f"cr_{relationship.slug}__{peer_side}"
                 self.fields[field_name] = relationship.to_form_field(side=side)
 
+                # if relationship.required in [
+                #     RelationshipSidesRequiredChoices.BOTH_SIDES_REQUIRED,
+                #     RelationshipSidesRequiredChoices.DESTINATION_SIDE_REQUIRED,
+                # ]:
+                #     self.fields[field_name].required = True
+
```

Comment:
This is currently commented out because I wanted to test the server-side validation before providing HTML validation.